### PR TITLE
GH-380 Add amCharts to MultiChart

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-380

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo Please Review

File: src/chart/MultiChart.js [hunk 1]
```diff
@@ -11,21 +11,27 @@
...
         { id: "GOOGLE_BAR", display: "Bar (Google)", widgetClass: "google_Bar" },
         { id: "GOOGLE_COLUMN", display: "Column (Google)", widgetClass: "google_Column" },
-        { id: "LINE", display: "Line", widgetClass: "chart_Line" },
         { id: "GOOGLE_LINE", display: "Line (Google)", widgetClass: "google_Line" },
-        { id: "C3_LINE", display: "Line (C3)", widgetClass: "c3chart_Line" },
+        { id: "C3_AREA", display: "Area (C3)", widgetClass: "c3chart_Area" },
         { id: "C3_BAR", display: "Bar (C3)", widgetClass: "c3chart_Bar" },
         { id: "C3_COLUMN", display: "Column (C3)", widgetClass: "c3chart_Column" },
+        { id: "C3_LINE", display: "Line (C3)", widgetClass: "c3chart_Line" },
+        { id: "C3_SCATTER", display: "Scatter (C3)", widgetClass: "c3chart_Scatter" },
```
Comment (mlzummo) [inline src/chart/MultiChart.js]: add am scatter too?

**Comment by mlzummo** [comment]
other than that looks good

**Comment by GordonSmith** [comment]
Scatter is not working with our regular data shapes...

**Pull request merged**